github_api_helper.js

```diff
@@ -98,22 +98,48 @@ function getConclusion (annotations) {
  * @param {import('probot').Context} context Probot context
  * @param {Number} runID check run identifier
  * @param {Object} output merged scan output
+ * @param {Number} annotationsPerPage Optional: number of annotations to send with one API call.
```

Comment:
How is this optional?  Will it work if I don't pass a value?  I don't see a default value

Reply:
I refactored the function because it was unclear what I was doing while using those variable names.
I must have been tired. :D

Reply:
There is a default value but it was not clear.

Reply:
Any plans to change this?

Reply:
I want to keep that parameter because I use it in my unit test for multiple annotations.
There as you can see: 
![image](https://user-images.githubusercontent.com/16246778/54866938-83df1300-4d83-11e9-937a-2929e2a2b227.png)

I want to send 2 annotations per API call which means that I will call two times github.checks.update which is want I want to test.